### PR TITLE
feat: add support for api url and openshift cluster configuration in the CR

### PR DESCRIPTION
**Pull request by JameelB**
Fix for https://github.com/fabric8-launcher/launcher-operator/issues/39

@edewit @gastaldi would you mind reviewing this please?

### CODE REVIEW

File: templates/fabric8-launcher.yaml [hunk 1]
```diff
@@ -26,7 +26,6 @@ parameters:
     should be created (ie. https://hostname[:port] or https://ipaddress[:port]).
     Leave the default value when targetting the same OpenShift/Minishift you are
     running this on
-  value: https://openshift.default.svc.cluster.local
```
Comment (gastaldi) [inline templates/fabric8-launcher.yaml]: The template is always copied from the launcher-application repository using the `update-template.sh`
 Can we keep this default value and set it to blank in the operator?

Comment (JameelB) [reply templates/fabric8-launcher.yaml]: changed the apiUrl to be an empty string by default :)

File: example/launcher_cr.yaml [hunk 2]
```diff
@@ -6,7 +6,44 @@ spec:
 
 ####### OpenShift Configuration
   openshift:
+    apiUrl: https://openshift.default.svc.cluster.local
     consoleUrl: #<the OpenShift Console Url (ex: 'https://192.168.64.4:8443')>#
+    clusters:
```
Comment (edewit) [inline example/launcher_cr.yaml]: maybe remove this, so that we keep the example as small as possible?

Comment (JameelB) [reply example/launcher_cr.yaml]: This has now been removed from the Launcher custom resource example :)

**Pull request merged**